test/validate/linux_validate.go

```diff
@@ -0,0 +1,232 @@
+package validate
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+
+	"github.com/Azure/azure-container-networking/cns"
+	restserver "github.com/Azure/azure-container-networking/cns/restserver"
+	k8sutils "github.com/Azure/azure-container-networking/test/internal/k8sutils"
+	"github.com/pkg/errors"
```

Comment:
No specific reason.
Since I was removing the functions out of the integration package. At that point using k8sutils as named import help a lot. (I find it easier to read)